docs/UserGuide.md

```diff
@@ -367,6 +379,20 @@ If your changes to the data file makes its format invalid, ReadyBakey will disca
 
 _Details coming soon ..._
 
+### Using Natural Dates
+
+In order to provide a quicker method of referring to near dates, ReadyBakey allows users to use natural dates to refer to the directly upcoming days of the week.
+The closest upcoming date that corresponds to the natural date's day input will be used. 
```

Comment:
Need to specify that it is sensitive to the time also?

Reply:
Nice catch, I'll add it in

Reply:
Updated this!